dm/config/task.go

```diff
@@ -510,7 +510,6 @@ func (c *TaskConfig) SubTaskConfigs(sources map[string]DBConfig) ([]*SubTaskConf
 		cfg.Mode = c.TaskMode
 		cfg.CaseSensitive = c.CaseSensitive
 		cfg.MetaSchema = c.MetaSchema
-		cfg.RemoveMeta = c.RemoveMeta
```

Comment:
How about removing `remove-meta` from `TaskConfig` and `SubTaskConfig`, and adding `remove-meta` as a command-line flag argument for `start-task`?

Reply:
there is an issue https://github.com/pingcap/dm/issues/496

Reply:
addressed